styles.css

```diff
@@ -0,0 +1,144 @@
+body {
+  background-image: url(images/backgroundimage.png);
+  background-repeat: no-repeat;
+}
```

Comment:
You can add `background-size: cover;` It will set it as cover.

Reply:
@RaoAkif quick question, all of the comments you sent are optional, so this should be approved isn't it? I am doing the changes, but I just would like to point out this since my next review will be the last one and you made just optional corrections. Thanks so much.

Reply:
@RaoAkif changes were done already, would you make the approval or another review request is needed? Thanks in advance!!

Reply:
@RaoAkif cover property does not work. It makes 3 times the background image.

Reply:
- These were not Optional. I made it optional because I realized that you are making some effort. But there are a lot of things to be improved.
- But you must do all the OPTIONAL changes that are requested. These are not difficult but they would refine your Portfolio very much. So do it as you do the REQUIRED ones. This is how you would get the habit of not ignoring little things.
- You have to Request another Review. I intentionally add a REQUIRED comment. So that you may not start with a poor landing page. So keep doing hard work. With time, you would develop this habit and become a good developer.
- Kindly test `cover` property again:
Before:
![image](https://user-images.githubusercontent.com/61361037/212438429-0c477bdc-c130-4bd1-82ee-2159ec9db6b8.png)

After:
![image](https://user-images.githubusercontent.com/61361037/212438401-94062433-9849-486d-85c2-184b7c64a1cc.png)

Reply:
One more thing that I noticed is:
![image](https://user-images.githubusercontent.com/61361037/212438783-c816b466-1d27-43ee-8138-de4263451fde.png)
These MUST be `li` tags. `ul > li`. Kindly correct this. Thanks

Reply:
All have been corrected. Another review was requested already. Thanks. @RaoAkif